website/docs/override-checking.md

```diff
@@ -41,6 +41,51 @@ Note that the **absence** of `abstract` or `overridable` does **not** mean that
 a method is never overridden. To declare that a method can never be overridden,
 look into [final methods](final.md).
 
+## A note on variance
+
+When overriding a method, the override must accept at all the same things that
+the parent method accepts, and return at most what the parent method returns but
+no more.
+
+This is very abstract so let's make it concrete with some examples:
```

Comment:
The only example we give is one with argument variance; should we also give an example with return value variance as well?